LayoutTests/accessibility/mac/text-input-marked-range.html

```diff
@@ -0,0 +1,24 @@
+<!DOCTYPE html>
+<html>
+<head>
+    <script src="../../resources/js-test-pre.js"></script>
```

Comment:
You can just import `<script src="../../resources/js-test.js"></script>` now, no need to import pre/post separately.

Reply:
Thanks! I didn't know that.